p1-staking-pool/src/constants.rs

```diff
@@ -9,10 +9,13 @@ pub const GAS_FOR_FT_TRANSFER: Gas = 10_000_000_000_000;
 pub const NO_DEPOSIT: Balance = 0;
 pub const ONE_YOCTO: Balance = 1;
 
-pub const NANO:u64 = 1_000_000_000;
+/// one second in nanoseconds
+pub const SECOND: u64 = 1_000_000_000;
+/// epoch = 1 minute
+pub const EPOCH: u64 = 60 * SECOND;
```

Comment:
please let's not use the word "EPOCH" because it has a definitive meaning in NEAR
we can use "NANOSECONDS_PERIOD" instead of "EPOCH" here, because that's what this constant is storing

Reply:
`NANOSECONDS_PERIOD` sounds weird. Everyone calls this thing EPOCH. In NEAR, EPOCH is also a time cycle when validation switch and staking rewards occurs. 

If you don't like EPOCH, then how about ROUND?

Reply:
ok